src/pages/verify/e-ids/swedish-bankid.mdx

```diff
@@ -60,6 +60,142 @@ For more information on how to configure your device for test and setup test use
+  
+  The caller provides a hint that can be used to send an authentication request to the caller's authentication device, typically a phone. The user can then approve the request and complete the authentication process as usual. 
+
+  CIBA is an extension to OpenID Connect. However, unlike other OpenID Connect flows, there is a direct communication between the Relying Party and the OpenID Provider, without redirects through the user's browser.
+
+</Highlight>
+
+### Implementing BankID Phone Authentication
+
+The CIBA flow is only available to traditional server-based web applications (confidential clients) and requires client validation for authentication requests. We'll use [private key JWTs](/verify/guides/privatekey-jwt/) for authenticaiton in the example below. An alternative (*less secure*) approach is to use `client_secret` for authentication.
+
+1. Make an `HTTP POST` request to the `/ciba/bc-authorize` endpoint:
+
+```javascript
+# Replace `$client_assertion` with your JWT assertion
+# Replace `YOUR_DOMAIN.criipto.id` with your Criipto domain
+# Replace `SSN` with a valid Swedish BankID SSN
+# Replace `BINDING_MESSAGE` with the text that will be presented to the end user
+# `callInitiator` must be either "RP"("Relying Party", e.g. a bank agent) or "user", depending on who initiated the call.
+
+HTTP POST https://YOUR_DOMAIN.criipto.id/ciba/bc-authorize
+Content-Type: application/x-www-form-urlencoded
+
+scope=openid
+&callInitiator=RP
+&login_hint=sub:ssn:SSN
+&acr_values=urn:grn:authn:se:bankid
+&binding_message=BINDING_MESSAGE
+&client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+&client_assertion=$client_assertion
+```
+
+2. A successful response will look something like this:
+
+```javascript
+{ 
+  "auth_req_id" : "3857f8ff-21b9-48ae-a732-a3bd8128a7ae",
+  "expires_in" : 120 
+}
+```
+
+3. Poll the token endpoint (`/oauth2/token`) to get a response with an ID token.  
+   The `auth_req_id` is used to construct a poll request on the token endpoint:
+
+```javascript
+# Replace `AUTH_REQ_ID` with the id returned by authentication response
+# Replace `$client_assertion` with your JWT assertion
+
+HTTP POST https://YOUR_DOMAIN.criipto.id/oauth2/token
+Content-Type: application/x-www-form-urlencoded
+
+auth_req_id=AUTH_REQ_ID
+&grant_type=urn:openid:params:grant-type:ciba
+&client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+&client_assertion=$client_assertion
+```
+
+**Polling Frequency**
+
+Bear in mind, that the authentication process requires interacting multiple times with the end-user, so the identification process could take some time. Consequently, the token endpoint should be polled every 5 seconds.
+
+4. The result of a poll request will depend on the state of the process:
```

Comment:
This `4` seems kind of misplaced

Reply:
due to the Polling fequency header, perhaps it should be formatted more as an form of "aside", perhaps using italics.